docs/examples/volume-snapshot/restoresession.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: stash.appscode.com/v1beta1
+kind: RestoreSession
+metadata:
+  name: demo-restore-from-volumesnapshot
+  namespace: demo
+spec:
+  volumeClaimTemplates:
```

Comment:
How do you know when to use restic vs VolumeSnapshot?

Reply:
volumeClaimTemplate will be used only for VolumeSnapshot.

Should we add driver here?